Nika_modules/FASTQ_tools.py

```diff
@@ -0,0 +1,28 @@
+dna_nucleotide_alphabet = {'A', 'T', 'G', 'C'}
+
+
+def filter_gc(seq, gc_bounds):
+    for nuc in seq:
+        if nuc not in dna_nucleotide_alphabet:
+            raise ValueError(f"{nuc} this is not a nucleotide in your sequence")
+    gc_content = (seq.count('G') + seq.count('C')) / len(seq) * 100
+    if isinstance(gc_bounds, int):
+        return gc_content <= gc_bounds
+    elif isinstance(gc_bounds, float):
+        return gc_content <= gc_bounds
+    elif isinstance(gc_bounds, tuple):
+        return gc_bounds[0] <= gc_content <= gc_bounds[1]
+
```

Comment:
В целом ок, но ты так получается расщепила линую событий на 3 ветки которые кажется очень похожи между собой. Я бы свел все к одной:
```suggestion
    if isinstance(gc_bounds, int) or isinstance(gc_bounds, float):
        gc_bounds = (0, gc_bounds)
        
    return gc_bounds[0] <= gc_content <= gc_bounds[1]
```
Но это так, кому как больше нравится)